jobs/JEVS_GLOBAL_ENS_PREP

```diff
@@ -38,16 +38,17 @@ export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/${ccpa_ver})}
 export COMINcmce=$COMINnaefs
 export COMINprepbufr=${COMINprepbufr:-$(compath.py $envir/com/obsproc/${obsproc_ver})}
-export COMINsnow=${DCOMIN}
-export COMINosi_saf=${COMINosi_saf:-$DCOMROOT}
-export COMINsst=${COMINsst:-$DCOMROOT} 
+export DCOMINsnow=${DCOMIN}
+export DCOMINosi_saf=${DCOMINosi_saf:-$DCOMROOT}
+export DCOMINsst=${DCOMINsst:-$DCOMROOT} 
 
 export COMINgefs_bc=${COMINgefs_bc:-$COMINnaefs}
-export COMINcmce_bc=${DCOMIN}
-export COMIN_gefs=${COMIN_gefs:-${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs}
-export COMIN_gefs_bc=${COMIN_gefs_bc:-${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs_bc}
-export COMIN_cmce=${COMIN_cmce:-${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce}
-export COMIN_cmce_bc=${COMIN_cmce_bc:-${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce_bc}
+export DCOMINcmce_bc=${DCOMIN}
+export EVS_gefs=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs
+export EVS_gefs_bc=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs_bc
+export EVS_cmce=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce
+export EVS_cmce_bc=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce_bc
```

Comment:
@BinbinZhou-NOAA It's surprising to me that we need EVSIN variables in the prep step of global_ens. I imagine that this is because you first need to gather some files from /com (in the previous COMIN variables), do some work on those files, save them, and then you need to call on those modified files for a subsequent prep step. Is that right?

If that's correct, please update these variables to be to start with EVS**IN** and not use an underscore:

```
export EVSINgefs=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs
export EVSINgefs_bc=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/gefs_bc
export EVSINcmce=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce
export EVSINcmce_bc=${COMIN}/${STEP}/${COMPONENT}/atmos.${INITDATE}/cmce_bc
```

CCing @PerryShafran-NOAA, @ShelleyMelchior-NOAA, and @StevenSimon-NOAA , who I would like to confirm this.

Reply:
My understanding is that these environmental variables flow into an ush file that grabs model and obs data for headline prep  /lfs/h2/emc/vpppg/noscrub/emc.vpppg/EVS/ush/global_ens/evs_get_gens_headline_data.sh.

Reply:
@StevenSimon-NOAA My question was really: "What are these EVS_gefs variables pointing to?" It is not clear what that are pointing to from the current name. I believe that the variables are pointing to some global_ens files that have been pre-processed as part of EVS, and are therefore in EVSINgefs, etc. Is that correct?

Reply:
That is not clear, but it appears so.